### PR TITLE
Add blue ocean

### PR DESCRIPTION
**Pull request by patprodrigues**
add new recipe

### CODE REVIEW

**Comment by patprodrigues** [comment]
@alfg Hi just to check if someone can check and approve this.

File: bue-ocean.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ],
+    "directions": [
+        "Add the water.",
+        "Add the sea salt to the water.",
+        "Incorporate the ice to the salt water."
+    ],
+    "image": "water.jpg",
+    "keywords": [
+        "glass of water",
+        "..."
```
Comment (alfg) [inline bue-ocean.json:30]: What is this for?

**Review by alfg — CHANGES_REQUESTED** [review]
Please resize image and re-submit.

File: bue-ocean.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Blue Ocean",
+    "description": "water based drink",
+    "github": "<patprodrigues",
```
Comment (alfg) [inline bue-ocean.json:4]: Just the username here.

**Comment by patprodrigues** [comment]
@alfg fixed, thank you.

**Pull request closed without merging**